public/components/add-modules-data/guides/office.js

```diff
@@ -0,0 +1,181 @@
+/*
+* Wazuh app - Amazon Web Services interactive extension guide
```

Comment:
I suggest changing Amazon Web Services to Office 365

Reply:
Done